### ISSUE TITLE
Move NCT into Alumni section and update copy

### ISSUE THREAD

**Issue by ToriCAST** [TechforgoodCAST/cast-website#170]
Can we please update copy to follow the format of the other pop ups: 

For 60 years, NCT has been supporting new and expectant parents. As the largest parenting charity in the UK, we share knowledge, make friends and  come together to campaign for change. Yet we know that many more people could benefit from NCT than do at present, and we know we can do more to support parents postnatally.

Our vision is of a world in which no parent is isolated and all parents are supported to build a strong society. It is a vision of a boundless network – of parents supporting parents, parenting together and parenting informed, whoever you are, wherever you’re from. Because we see that as a movement of passionate people, standing on our collective strength – our expertise, our scale and our brand – we have a determination to do more and go further.

And this is where Fuse comes in.....

NCT is going through a time of change and renewal, a time of re-discovery and innovation. My time on Fuse supports two of our key goals:
•	To strengthen our core work in antenatal education and infant feeding
•	To modernise our public image and increase our reach so that we are relevant and trusted by all parents wherever they live, wherever they are from.

The product we worked on is a modern and intuitive course booking process, initially designed for our antenatal courses but with potential to be rolled out to all of our services, both paid for and offered by our network of over 300 branches, run by volunteers.

It has great scope to evolve to be richer, enhancing the NCT journey and keeping parents with us throughout their first 1,000 days of parenthood.

"I've gained so much, both professionally and personally, from the Fuse accelerator. The methodology I've learnt is one that I know we can use more widely within NCT, across many areas of our charity. It's hugely exciting to know that this is just the beginning of our digital transformation, and that there is such scope for development and growth, with colleagues from Fuse supporting us as we move forwards."

Tamara Bavin

**Comment by eghale** [comment]
![nct_fuse_edited](https://user-images.githubusercontent.com/11066569/27232358-af4241e8-52ad-11e7-8eef-3aa7ba98d61a.jpg)

**Comment by ivan-unfolds** [comment]
@ToriCAST I have pushed changes to staging site (https://wearecast-staging.herokuapp.com). Could you have a look and if it is all okay I will push to live site.

**Comment by ToriCAST** [comment]
Hey @ivanmauricio thanks for looking into this! I've just spotted a couple of minor tweaks before we go live, if you don't mind: 

- Can we just switch over the hyphen to a colon por favor, on the following paragraph: 

NCT is going through a time of change and renewal, a time of re-discovery and innovation. My time on Fuse supports two of our key goals - 

- And merge the last 2 paragraphs before the quote, just to make it look a little less long! 

The product we worked on is a modern and intuitive course booking process, initially designed for our antenatal courses but with potential to be rolled out to all of our services, both paid for and offered by our network of over 300 branches, run by volunteers. It has great scope to evolve to be richer, enhancing the NCT journey and keeping parents with us throughout their first 1,000 days of parenthood.

Looks great otherwise! Thanks @eghale for the photo suggestions :)
